ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__nightly-4.14-loaded-upgrade-from-stable-4.12.yaml

```diff
@@ -0,0 +1,75 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-ld-414
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  initial:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.12.0-0
+        upper: 4.13.0-0
+  intermediate:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.13.0-0
+        upper: 4.14.0-0
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-loaded-upgrade-aws
+  cron: 0 12 * * 1
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:initial
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      CLEANUP_WHEN_FINISH: "false"
+      CONTROL_PLANE_INSTANCE_TYPE: m5.2xlarge
+      ENABLE_FORCE: "true"
+      MAX_UNAVAILABLE: "3"
+      WORKER_REPLICA_COUNT: "5"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre-stableinitial
+    - chain: workers-scale
+    - chain: openshift-qe-cluster-density
+    test:
+    - as: set-releases-release
```

Comment:
Is this required to be as a separate test in order to perform this command and specify dependencies?
"${RELEASE_IMAGE_INTERMEDIATE} ${RELEASE_IMAGE_LATEST}" > ${SHARED_DIR}/perfscale-override-upgrade

If yes, can we print something to the output of that test as part of the check? I see https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/40363/rehearse-40363-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-nightly-4.14-loaded-upgrade-from-stable-4.12-ocp-qe-perfscale-loaded-upgrade-aws/1671000929530286080/artifacts/ocp-qe-perfscale-loaded-upgrade-aws/set-releases-release/build-log.txt as blank. Can we print some output here instead of blank file?

Reply:
Also may be name something more intuitive? May be something like "set-intermediate-releases"?

Reply:
This step wasnt added as part of this PR. not sure how much we'll be able to rename this if its used in other workloads already

Reply:
oh sorry, nevermind. Thought this was a reusable step from step-registry. Think we can still rename

Reply:
Rename to set-intermediate-releases and adding print message